app/src/main/java/org/wikipedia/diff/ArticleEditDetailsFragment.kt

```diff
@@ -528,4 +462,17 @@ class ArticleEditDetailsFragment : Fragment(), WatchlistExpiryDialog.Callback, L
         setPlainText(requireContext(), null, uri)
         FeedbackUtil.showMessage(this, R.string.address_copied)
     }
+
+    companion object {
+        const val EXTRA_ARTICLE_TITLE = "articleTitle"
+        const val EXTRA_EDIT_REVISION_ID = "revisionId"
+        const val EXTRA_EDIT_LANGUAGE_CODE = "languageCode"
+
+        fun newInstance(articleTitle: String, revisionId: Long, languageCode: String): ArticleEditDetailsFragment {
+            val articleEditDetailsFragment = ArticleEditDetailsFragment()
+            articleEditDetailsFragment.arguments = bundleOf(EXTRA_ARTICLE_TITLE to articleTitle,
+                    EXTRA_EDIT_REVISION_ID to revisionId, EXTRA_EDIT_LANGUAGE_CODE to languageCode)
+            return articleEditDetailsFragment
```

Comment:
Would it be possible to use `return ArticleEditDetailsFragment().apply { }`, just like what we have done in other places?

Reply:
yep!